src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -0,0 +1,104 @@
+  (let [theme            (quo.theme/use-theme)
+        accounts         (parse-accounts (:accounts item))
+        default-keypair? (zero? index)
+        details          {:full-name (:name item)
+                          :address   (when default-keypair?
+                                       (utils/get-shortened-compressed-key compressed-key))}
+        on-press         (rn/use-callback
+                          (fn []
+                            (on-options-press
+                             (merge {:theme theme
+                                     :title (:full-name details)}
+                                    (if default-keypair?
+                                      {:type                :default-keypair
+                                       :description         (:address details)
+                                       :customization-color customization-color
+                                       :profile-picture     profile-picture}
+                                      {:type        :keypair
+                                       :icon-avatar :i/seed}))))
+                          [customization-color default-keypair? details profile-picture theme])]
+    [quo/keypair
+     {:blur?               false
+      :status-indicator    false
+      :stored              :on-device
+      :action              :options
+      :accounts            accounts
+      :customization-color customization-color
+      :container-style     style/keypair-container-style
+      :profile-picture     (when default-keypair? profile-picture)
+      :type                (if default-keypair? :default-keypair :other)
+      :on-options-press    on-press
+      :details             details}]))
+
+(defn view
+  []
+  (let [inset-top           (safe-area/get-top)
+        compressed-key      (rf/sub [:profile/compressed-key])
+        profile-picture     (rf/sub [:profile/image])
+        customization-color (rf/sub [:profile/customization-color])
+        keypairs            (rf/sub [:wallet/keypairs])]
+    [quo/overlay
+     {:type            :shell
+      :container-style (style/page-wrapper inset-top)}
+     [quo/page-nav
+      {:key        :header
+       :background :blur
+       :icon-name  :i/arrow-left
+       :on-press   navigate-back}]
+     [rn/view {:style style/title-container}
+      [quo/standard-title
+       {:title               (i18n/label :t/keypairs-and-accounts)
+        :accessibility-label :keypairs-and-accounts-header
+        :customization-color customization-color}]]
+     [rn/view {:style {:flex 1}}
+      [rn/flat-list
+       {:data                    keypairs
+        :render-fn               keypair
+        :render-data             {:profile-picture     profile-picture
+                                  :compressed-key      compressed-key
+                                  :customization-color customization-color}
+        :initial-num-to-render   1
+        :content-container-style {:padding-bottom 60}}]]]))
```

Comment:
Not sure if need this prop: https://reactnative.dev/docs/flatlist#initialnumtorender

It seems to be for managing how many will be batch rendered on the screen, but we're suppose to pass a number that would fill the entire screen. The default is `10`, so unless there's a performance reason for passing `1`, I'll probably remove this prop.